src/utilities/flexbox.scss

```diff
@@ -48,5 +49,9 @@
       flex-grow: 1;
       flex-basis: 0;
     }
+
+    .flex#{$variant}-order-1 { order: 1 !important; }
+    .flex#{$variant}-order-2 { order: 2 !important; }
+    .flex#{$variant}-order-none { order: inherit !important; }
```

Comment:
I guess we could use `order: 0` instead of `inherit ` to "reset" the order since that is the default value?